### ISSUE TITLE
xtensa-lx106-elf-gcc:  error while compiling this code.

### ISSUE THREAD

**Issue by Daanishh** [debsahu/PCF8574_8Relay#2]
I am using arduino IDE to code nodemcu / esp8266.

while compiling the code I am getting the following error,

xtensa-lx106-elf-gcc: error: libraries\DDUpdateUploadServer-master\DDUpdateUploadServer-master.a: No such file or directory

I have already downloaded all of the dependencies you have mentioned, I have tried few fixes but that does not seems to be working.

can you help me on this ?
Thanks

**Comment by debsahu** [comment]
Did you install this? https://github.com/debsahu/DDUpdateUploadServer

**Comment by Daanishh** [comment]
> Did you install this? https://github.com/debsahu/DDUpdateUploadServer

Yes, I have installed this library as well along with everything you have mentioned.
![image](https://user-images.githubusercontent.com/6799170/66939249-27637e00-f05c-11e9-8941-18b0f74b35aa.png)

**Comment by debsahu** [comment]
Please delete that library and reinstall. It compiles on PIO, I dont use Arduino IDE just for such reasons.

**Comment by Daanishh** [comment]
> Please delete that library and reinstall. It compiles on PIO, I dont use Arduino IDE just for such reasons.

Okay, thank you for replying quickly brother, 

I am getting an error on PIO as well. it is again related to the library  **DDUpdateUploadServer** you have provided.

![image](https://user-images.githubusercontent.com/6799170/67030876-c609e080-f129-11e9-8818-e74d937c1df7.png)

**Comment by debsahu** [comment]
This should fix PIO compilation issue. https://github.com/debsahu/PCF8574_8Relay/commit/2b9a959e36945a69a6dc5e97d32d8def5e5eed6b
